ush/parsing_namelists_FV3.sh

```diff
@@ -328,6 +328,7 @@ cat >> input.nml <<EOF
   cnvcld       = ${cnvcld:-".true."}
   imfshalcnv   = ${imfshalcnv:-"2"}
   imfdeepcnv   = ${imfdeepcnv:-"2"}
+  progsigma    = ${progsigma:-".true."} 
```

Comment:
Is this going to work if using GFDL MP?

Reply:
@lisa-bengtsson can you answer this?

Reply:
Currently no. It will need a separate field table for GFDL microphysics (and other MP schemes such as Morrison Gettleman).

Reply:
That's already happening, but this needs to not be written to the FV3 namelist outside of Thompson then. Preferably in an existing if-block for Thompson if there is one.

Reply:
Could it be set default .false. in this routine and then .true. in config.fcst under a "HR1" flag or "Thompson" flag?

Reply:
@lisa-bengtsson we have the new field tables for GFDL-MP so this "should be" okay now, correct?

Reply:
Yes, this should be OK now. Thank you.